pkg/lib/cockpit-components-listing-panel.scss

```diff
@@ -8,6 +8,9 @@
     display: flex;
     align-items: flex-end;
     padding-right: var(--pf-global--spacer--lg);
+    gap: var(--pf-global--spacer--sm);
+    flex-wrap: wrap;
+    justify-content: space-between;
```

Comment:
Why is this not needed?

In smaller sizes, especially with longer text from other languages, there isn't enough horizontal room to have the tabs and the weird date on the same line. This forces the tabs and the text to wrap and the UI breaks.

If you'd rather move the date to the line instead, then we don't need to come up with solutions for wrapping here. But if we don't do that, then we need to adjust the UI so it doesn't break.

Reply:
This CSS just says:

1. Make sure there's a gap between the items (so the text from both elements doesn't touch)
2. Wrap it when it won't fit otherwise
3. Space them out, so they're left and right aligned (when they're not wrapping)... and when they wrap, it doesn't apply, so they're both left aligned by default.

Reply:
Moved it into a Flex PF4 component in the commit I suggested above.